friends/src/components/Friends/FriendCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import EditFriend from '../Forms/EditFriend';
+
+const FriendCard = props => {
+    return (
+        <table>
```

Comment:
any particular reason you chose to use a table here?

Reply:
Cause you'd mentioned using a table for data awhile back in one of my code reviews ;). Personally I hate them (nightmares from table layouts). Could of been a group of divs.